css/style.css

```diff
@@ -0,0 +1,215 @@
+/* general */
+body {
+  min-width: 1366;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+h2 {
+  padding: 0;
+  margin: 0;
+}
+
+p {
+  padding: 0;
+  margin: 0;
+}
+
+/* typography */
+.primary-font-family {
+  font-family: "Poppins", sans-serif;
+}
+
+.logo-font {
+  font-family: "Poppins", sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  color: #6070ff;
+}
+
+.banner-header {
+  font-family: "Poppins", sans-serif;
+  font-size: 38px;
+  line-height: 52px;
+  font-weight: 700;
+  color: #172b4d;
+}
+
+.body-text-1 {
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  line-height: 52px;
+  font-weight: 300;
+  color: #172b4d;
+}
+
+.body-text-2 {
+  font-family: "Poppins", sans-serif;
+  font-size: 15px;
+  line-height: 24px;
+  font-weight: 300;
+  color: #172b4d;
+}
```

Comment:
- [ ] Kindly make sure your body text has the same **size** and **color** as the one used on the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) design.
I know you can do it :muscle: and this will make your design matches perfectly the one from Figma.